pkg/resource/events-logger-config/grafana-agent/test/events-logger-config.grafanaagent.MC.yaml

```diff
@@ -19,7 +20,7 @@ grafana-agent:
         
         remote.kubernetes.secret "credentials" {
         	namespace = "kube-system"
-        	name = "test-installation-grafana-agent-secret"
+        	name = "test-installation-events-logger-secret"
```

Comment:
This is wrong though as the secret for grafana agent should bé test-installation-grafana-agenr

Reply:
I agree, I also found it weird when the `go test --update` made this change. Somehow, it wasn't working with the previous name which doesn't make sense because the `GetEventsLoggerSecretName` function returns `fmt.Sprintf("%s-%s", lc.GetClusterName(), grafanaAgentSecretName)` (with `grafanaAgentSecretName = grafana-agent-secret"`) when grafana-agent is the events-logger

Reply:
So it's because the test loggedcluster did not set the SetKubeEventsLogger() https://github.com/giantswarm/logging-operator/blob/4d9f9d075364a118efa522f3e8a56ef65d6fbaa6/pkg/resource/events-logger-config/grafana-agent-config_test.go#L56 so you would need to set the KubeEventsLogger to GrafanaAgent in the test

Reply:
Fixed.